.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Set up Ruby 2.6
```

Comment:
This is now wrong

Reply:
ugh right - good catch. Not sure why the version needs to be in the name anyway